### ISSUE TITLE
[BUG] Modus Table - Accessibility Issue - It displays as 1 Violation

### ISSUE THREAD

**Issue by HarshithaNous** [trimble-oss/modus-web-components#1609]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

Modus Table - Accessibility Issue - It displays as 1 Violation 

Please refer the below screenshot:
<img width="957" alt="image" src="https://github.com/trimble-oss/modus-web-components/assets/129380495/205909b6-4efc-496c-8b0d-07b0df5a436a">

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

High

### What product/project are you using Modus Components for ?

QA

### What is your team/division name ?

QA

### Are you willing to contribute ?

None
